deployment/ccip/changeset/test_helpers.go

```diff
@@ -307,10 +314,8 @@ func NewMemoryEnvironmentWithJobsAndContracts(t *testing.T, lggr logger.Logger,
 			SourceMessageTransmitterAddr: state.Chains[chain].MockUSDCTransmitter.Address().String(),
 		}
 	}
-	for _, chain := range allChains {
-		timelocksPerChain[chain] = state.Chains[chain].Timelock
-		ocrParams[chain] = DefaultOCRParams(e.FeedChainSel, nil, nil)
-	}
+	require.NotNil(t, state.Chains[e.FeedChainSel].LinkToken)
+	require.NotNil(t, state.Chains[e.FeedChainSel].Weth9)
```

Comment:
Do we need this as we are doing similar validation for all the chains?
https://github.com/smartcontractkit/chainlink/pull/15426/files#diff-647d1f54557a9137c5711f752ef36c682ac55c1af4883782fda38622ed6b0a70R367-R379